workflow/generate_baseline_report.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env pythona
+import os
+import shutil
+import subprocess
+
+from git import Repo
+from asapp.common import config
+from workflow.evaluate_model import run as evaluate_model
+
+from workflow.paths import *
+
+class GenerateBaselineReport:
+
+    def __init__(self, config):
+        self._config = config['metrics']['metric'][0]
+        self._baseline = self._config['baseline']
+        self._releases = self._config['releases']
+
+    def checkout_model_repos(self, release):
+        gasapp = Repo(ASAPP_SRS_ROOT)
+        gasapp.git.checkout(release)
+
+        gprodml = Repo(ASAPP_PRODML_ROOT)
+        gprodml.git.checkout(release)
+
+        gmleng = Repo(ASAPP_MLENG_ROOT)
+        gmleng.git.checkout(release)
+
+        gcomcast = Repo(ASAPP_COMCAST_SRS_ROOT)
+        gcomcast.git.checkout(release)
+
+        # get the business
```

Comment:
Oh man. More descriptive comment, please :). Source: gave me huge headache that one night.